src/web/Button.tsx

```diff
@@ -41,11 +41,12 @@ const _styles = {
 const _longPressTime = 1000;
 const _defaultAccessibilityTrait = Types.AccessibilityTrait.Button;
 
-let _isNavigatingWithKeyboard = false;
+let _isNavigatingWithKeyboard: boolean;
```

Comment:
Why are we keeping redundant copies of this piece of state in a bunch of modules? It's cheap to access the "truth" version by calling UserInterface.isNavigatingWithKeyboard whenever we need it.

Reply:
Yeah, it's treated very carefully, with caching, suppressing of repeated events, etc., a little overdone.
I'll simplify.